File: dotnet/src/Model/Enemy/Units/EWorker.cs
using System;

namespace PlanetCS1 {
    class EWorker : Worker {
        public EWorker(Location location) {
            this.loc = location;
            this.use = true;
            this.drive = true;
            this.owner = 2;
            this.icon = "eW";
        }
    }
}

File: typescript/src/PlanetTS1/Model/Enemy/Units/EWorker.ts
import { ALocation as ALocation } from "../../Utilities"

export class EWorker extends AWorker {
    loc : ALocation
    use : boolean
    drive : boolean
    owner : number
    icon : string
    constructor(location : ALocation) {
        super()
        this.loc = location
        this.use = true
        this.drive = true
        this.owner = 2
        this.icon = "eW"
    }
}